internal/bot/config.go

```diff
@@ -13,11 +13,53 @@ import (
 
 // Config struct models the bot.yml configuration file
 type Config struct {
-	Name       string           `mapstructure:"bot_name"`
-	Extensions extension.Config `mapstructure:"extensions"`
-	Store      fsm.StoreConfig  `mapstructure:"store"`
-	Port       int              `mapstructure:"port"`
-	Path       string
+	Name         string           `mapstructure:"bot_name"`
+	Extensions   extension.Config `mapstructure:"extensions"`
+	Store        fsm.StoreConfig  `mapstructure:"store"`
+	Port         int              `mapstructure:"port"`
+	Path         string
+	Conversation struct {
```

Comment:
Added a new `Conversation` setting. This allows us to configure how Chatto replies to conversations..